diff --git a/init/main.c b/init/main.c
index 5e13c544bbf4..f8885233d6ae 100644
--- a/init/main.c
+++ b/init/main.c
@@ -1067,7 +1067,7 @@ static int __ref kernel_init(void *unused)
 	mark_readonly();
 	system_state = SYSTEM_RUNNING;
 	numa_default_policy();
-	printk("##### Suchita Singh Kanthwal UFID:9855-9781  Hey!! What's up? #####");
+	printk("##### Suchita Singh Kanthwal (Exercise 2) #####");
 	rcu_end_inkernel_boot();
 
 	if (ramdisk_execute_command) {
